src/main/java/ruslan/shastkiv/bookstore/service/user/UserServiceImpl.java

```diff
@@ -22,7 +24,9 @@ public UserDto register(UserRegistrationRequestDto registrationRequestDto) {
                     + registrationRequestDto.email() + "] already registered!"
             );
         }
-        User savedUser = userRepository.save(userMapper.toModel(registrationRequestDto));
+        User userFromRequest = userMapper.toModel(registrationRequestDto);
+        userFromRequest.setPassword(passwordEncoder.encode(registrationRequestDto.password()));
+        User savedUser = userRepository.save(userFromRequest);
```

Comment:
set USER role to user

```suggestion
        User user = userMapper.toModel(registrationRequestDto);
        user.setPassword(passwordEncoder.encode(registrationRequestDto.password()));
        userRepository.save(user);
```